.ci/node7/Dockerfile.linux

```diff
@@ -0,0 +1,9 @@
+FROM node:7.10.1
+
+RUN apt-get update && \
```

Comment:
Nice! This means we now have a docker image that's a part of CI that is guaranteed to work with pptr.
This will be very helpful for everybody who runs puppeteer in a docker

cc @ebidel

Reply:
If you are interested we can add [Container Builder](https://medium.com/cirruslabs/introducing-container-builder-for-cirrus-ci-80b9234f007) later on to publish such official images to Docker Hub the [same way we do it for Flutter](https://github.com/cirruslabs/docker-images-flutter).

Reply:
@fkorotkov this would be great. Let's give cirrus some time and see how it goes. If it proves to be reliable, we'll be happy to publish puppeteer-compatible docker images.